### ISSUE TITLE
Rule proposal: indent props

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#15]
A rule that enforces that props on their own lines are indented properly might be nice.

Good:

```jsx
<MyComponent
  firstProp="value"
  secondProp="value"
/>
```

Bad:

```jsx
<MyComponent
firstProp="value"
secondProp="value"
/>
```

**Comment by andreypopp** [comment]
Shouldn't `/>` be indented as well?
```
<MyComponent
  firstProp="value"
  secondProp="value"
  />
```

**Comment by lencioni** [comment]
@andreypopp perhaps that should be handled by a separate rule? In my environment, Vim automatically dedents the `/>` as soon as I type it, and it is nice to go with the flow. Plus, I think it makes some sense if you think of these angle brackets like you would think of curlies in normal JS.
